doc/en/api/1.0.0/gwcfilterupdate.yaml

```diff
@@ -0,0 +1,46 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache Filter Update
+  description: The REST API for Updating GWC filter
+schemes:
+- http
+host: localhost:8080
+basePath: /geoserver/gwc/rest
+paths:
+  /filter/{filterName}/update/{updateType}:
+    post:
+      consumes:
+        - application/xml
+        - application/zip
+      operationId: filterUpdatePost
+      summary: Updates a given layer filter by way of xml or zip file.
+      description: Restfully updates the given filterName with parameters provided in the xml or zip
+      parameters:
+        - name: filterName
+          in: path
+          required: true
+          description: The name of the filter to update.
+          type: string
+        - name: updateType
+          in: path
+          required: true
+          description: One of 'zip' or 'xml'
```

Comment:
We can make this an enum of 'zip' or 'xml'.

See: https://swagger.io/docs/specification/data-models/enums/

```
schema:
  type: string
  enum: [zip, xml]
```

Reply:
Updated along with other similar parameters to use enums.